tdrs-backend/tdpservice/users/migrations/0041_users_digit_group_add_datafile_permission.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.2.5 on 2021-08-16 14:10
+from django.contrib.auth.models import Group
+from django.db import migrations
+
+from tdpservice.users.permissions import (
+    add_permissions_q,
+    get_permission_ids_for_model,
+    view_permissions_q
+)
+
+
+def set_digit_team_permissions(apps, schema_editor):
+    """Set relevant Group Permissions for DIGIT Team group."""
+    digit = (
+        apps.get_model('auth', 'Group').objects.get(name='DIGIT Team')
+    )
+
+    stt_permissions = get_permission_ids_for_model(
+        'stts',
+        'stt',
+        filters=[view_permissions_q]
+    )
+
+
+    datafile_permissions = get_permission_ids_for_model(
+        'data_files',
+        'datafile',
+        filters=[view_permissions_q, add_permissions_q]
+    )
+
+    # Assign model permissions
+    digit.permissions.add(*datafile_permissions, *stt_permissions)
```

Comment:
was it necessary to add object-level permissions? i don't recall ever using these, we normally just use the group/role assignments

Reply:
Not really, but I still decided to do it to match permissions with frontend. I think we should have let the backend manage the permission as whole, but I guess this needs some rework.